testing/consul/APKBUILD

```diff
@@ -61,9 +37,13 @@ prepare() {
 
 build() {
 	cd "$_builddir"/$pkgname || return 1
-	export GOPATH="$srcdir"
-	go build -v -o bin/consul \
-		-ldflags "-X main.GitDescribe $pkgver" || return 1
+
+	mkdir -p "${_builddir}/src/github.com/hashicorp"
+	ln -s "${_builddir}" "${_builddir}/src/github.com/hashicorp/consul"
+	cd "${_builddir}/src/github.com/hashicorp/consul"
+
+	GOPATH="$_builddir" go build -v -o bin/consul \
+		-ldflags "-X main.GitDescribe=$pkgver" || return 1
```

Comment:
@7heo They are. saw this for vault, consul, consul-template (still need to PR that in separately)